tools/tools_common.h

```diff
@@ -440,7 +455,7 @@ void signal_handler(int sig)
 
 void setupSignals()
 {
-#ifndef _MSC_VER
+#ifndef DISABLE_SIGNALS
```

Comment:
move this change to a separate PR (maybe with the getopt changes)

Reply:
Underway!

Reply:
Done in #756